library/src/CMakeLists.txt

```diff
@@ -80,20 +80,11 @@ if( BUILD_WITH_TENSILE )
     target_link_libraries( Tensile PRIVATE hip::device )
   endif()
 
-  if( CMAKE_CXX_COMPILER MATCHES ".*/hcc$" )
-    # Remove following when hcc is fixed; hcc emits following spurious warning ROCm v1.6.1
-    # "clang-5.0: warning: argument unused during compilation: '-isystem /opt/rocm/include'"
-    # target_compile_options( Tensile PRIVATE -Wno-unused-command-line-argument -fno-gpu-rdc -Wno-deprecated-declarations)
+  if( CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" )
```

Comment:
This was a breaking change alone so expect you may break 3.5 on some gpus with this.

Reply:
That wouldn't surprise me. I really need CI to run this. Haven't even tested this on other distros